<launch>
<!-- Launch file for Mapping -->

	<arg name="rviz" default="true" />

	<!-- <rosparam command="load" file="$(find fast_lio3)/config/avia.yaml" /> -->
	<!-- <rosparam command="load" file="$(find fast_lio3)/config/horizon.yaml" /> -->
    <!-- <rosparam command="load" file="$(find fast_lio3)/config/velodyne.yaml" /> -->
    <!-- <rosparam command="load" file="$(find fast_lio3)/config/ouster64.yaml" /> -->
    <rosparam command="load" file="$(find fast_lio3)/config/mapping_dev.yaml" />

	<param name="localization_mode" type="bool" value="false" />
    <node pkg="fast_lio3" type="fastlio_ros1" name="system" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio3)/rviz_cfg/mapping.rviz" />
	</group>

</launch>
